web/src/compose_ui.ts

```diff
@@ -202,7 +202,12 @@ export function maybe_show_scrolling_formatting_buttons(container_selector: stri
 function get_focus_area(opts: ComposeTriggeredOptions): string {
     // Set focus to "Topic" when narrowed to a stream+topic
     // and "Start new conversation" button clicked.
-    if (opts.message_type === "stream" && opts.stream_id && !opts.topic && realm.realm_mandatory_topics) {
+    if (
+        opts.message_type === "stream" &&
+        opts.stream_id &&
+        !opts.topic &&
+        realm.realm_mandatory_topics
+    ) {
```

Comment:
Looks like this belongs to the earlier commit 79ba615c7924476052e7de8eb70df9c11210da30